metrics.go

```diff
@@ -31,6 +31,12 @@ var (
 		[]string{"channel", "status"}, nil,
 	)
 
+	channelDeployedRevisionDeltas = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "channel_undeployed_revisions"),
```

Comment:
These metrics are all per-channel, without `channel_` as the prefix. Is there a reason that this metric in particular deserves that prefix?